packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -109,8 +112,9 @@ export async function getPortalsTradeQuote(
     const outputToken = `${portalsNetwork}:${buyAssetAddress}`
 
     // Attempt fetching a quote with validation enabled to leverage upstream gasLimit estimate
-    const portalsTradeOrderResponse = await fetchPortalsTradeOrder({
-      sender: sendAddress,
+    const portalsTradeQuoteResponse = await fetchPortalsTradeOrder({
```

Comment:
seems like we could be smarter with these quotes if we don't have a wallet to save two guranteed failed requests. could probably wait until part 2 follow up though.

Reply:
43fe29b9a8